### ISSUE TITLE
Backup for jcache?

### ISSUE THREAD

**Issue by yjanezou** [hazelcast/hazelcast#5881]
For IMap, there is a default 1 backup for the data, and it is also configurable. Is there similar setting for the ICache? if one node is down, would all the data in that node be lost?

**Comment by yjanezou** [comment]
is that I have to set the backup-count for the default cache

<cache name="default">
     <key-type class-name="java.lang.Object" />
     <backup-count>1</backup-count>
....

from the schema file, it seems the default value is 0. So if i set the configuration for "default" cache, it will apply to all other caches created from client side? Could client side configure this (I don't see such method in the MutableConfiguration class which is used by the client to create customized cache)?

**Comment by noctarius** [comment]
This is a mistake in the XML schema, the default is 1. You can find it in com.hazelcast.config.CacheSimpleConfig#DEFAULT_BACKUP_COUNT

If you like to contribute you might want to fix the XSD and create a pull-request :-)

**Comment by yjanezou** [comment]
for my testing, I used the server.sh to start 2 nodes at local, then I set up an test app to write data into the cache. It seems I have to modify the hazelcast.xml file (under same directory with the server.sh: hazelcast-3.5/bin) to add "default" cache element which I put backup-count as 1. Then the test with repeatedly bringing down a node or the other looks fine without cache miss. But testing with the original hazelcast.xml file shows there is no back up at all, say, bring down node 1, app writes all data to node 2, then bring back node 1 (I saw "Re-partitioning cluster data... Migration queue size: 135" output from node 2),  after it said "All migration tasks have been completed, queues are empty", I bring down node 2, here I assume node 1 should have backup data already. but the app query would fail due to cache miss.

Is there a bug?
